lib/tbot/config/testdata/TestSPIFFEWorkloadAPIService_YAML/full.golden

```diff
@@ -31,3 +31,5 @@ attestors:
       skip_verify: true
       anonymous: true
 jwt_svid_ttl: 5m0s
+credential_ttl: 1m0s
```

Comment:
This is one slight quirk of having `credential_ttl` on all of the outputs. We _could_ deprecate `jwt_svid_ttl` but given this whole output will be replaced with `workload-identity-jwt` I'm tempted to leave this as-is.

Reply:
👍  Happy to leave this as is for now since we'll be deprecating this soon.